.github/workflows/ci.yaml

```diff
@@ -88,16 +94,21 @@ jobs:
 
       - name: "Build SpiderLightning and Slight"
         run: make build
-        if: ${{ fromJSON(matrix.config.os == 'macos-latest') }}
-        env:
-          OPENSSL_ROOT_DIR: /usr/local/opt/openssl
+        if: ${{ fromJSON(matrix.config.os == 'macos-latest') && 
+          fromJSON(matrix.config.arch == 'amd64') }}
+
+      - name: "Build SpiderLightning and Slight"
+        run: cargo build --manifest-path ./slight/Cargo.toml --release ${{ matrix.config.extraArgs }}
```

Comment:
same comment as below

Reply:
~didn't want to do it 'cause of the github action specific argument

Reply:
sounds reasonable. My only concern is that if we refactor the code in the future that requires us to change the `cargo build` command to some other path, we might forget about changing the command in the github action.